.github/workflows/integration-tests-reusable.yml

```diff
@@ -66,10 +75,16 @@ jobs:
       run: |
         sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 60
     - uses: actions/setup-python@v1
-    - name: Set up JDK 1.8
+    - name: Set up oldest supported JDK
+      if: matrix.target-language-version == "oldest"
```

Comment:
Turns out double quotes are a syntax error (https://github.com/dafny-lang/dafny/actions/runs/2756979457) - string literals have to use single quotes: https://docs.github.com/en/actions/learn-github-actions/expressions#literals